.gitlab-ci.yml

```diff
@@ -54,7 +54,15 @@ stages:
     - make -j$(grep "core id" /proc/cpuinfo | sort -u | wc -l)
   artifacts:
     paths:
-      - build/
```

Comment:
Sorry, I might have misunderstood, but do we need the `build/doc/` artifacts for the documentation deployment or rather just to download ?

Reply:
These artifacts are used only by the tests. That is how the test step can reuse whatever was done in the build one.